Resources/views/Collector/swiftmailer.html.twig

```diff
@@ -175,6 +175,27 @@
                             </pre>
                         </div>
                     {% endfor %}
+
+                    {% if message.children %}
```

Comment:
Actually, a children can be something else than an attachment (according to the interfaces), but I don't really what it can really be...